main.tf

```diff
@@ -166,3 +166,23 @@ resource "aws_iam_role_policy_attachment" "this" {
   role       = aws_iam_role.this[0].name
   policy_arn = var.policy_arn
 }
+
+## Parameter Group
```

Comment:
Since we are adding a parameter group, should we treat this release as a feature release?

Reply:
@ndegwajohn  I had thought so but since this feature resolves one of the security alerts I thought it is good for it to be a fix

Reply:
at the moment I have tried rename the branch which closes the PR. We want to retain the comments here @ndegwajohn